frontend/packages/console-dynamic-plugin-sdk/docs/api.md

```diff
@@ -394,10 +404,48 @@ const HomePage: React.FC = (props) => {
 
 ### Summary
 
-[For more details please refer the implementation](https://github.com/openshift/console/tree/release-4.11/frontend/public/components/factory/Table/VirtualizedTable.tsx)
+A component for making virtualized tables
 
 
 
+### Example
+
+
+```ts
+const MachineList: React.FC<MachineListProps> = (props) => {
+  return (
+    <VirtualizedTable<MachineKind>
+     {...props}
+     aria-label='Machines'
+     columns={getMachineColumns}
+     Row={getMachineTableRow}
+    />
+  );
+}
+```
+
+
+
+
+
+### Parameters
+
+| Parameter Name | Description |
+| -------------- | ----------- |
+| `data` | data for table |
+| `loaded` | flag indicating data is loaded |
+| `loadError` | error object if issue loading data |
+| `columns` | column setup |
+| `Row` | row setup |
+| `unfilteredData` | original data without filter |
+| `NoDataEmptyMsg` | (optional) no data empty message component |
+| `EmptyMsg` | (optional) empty message component |
+| `scrollNode` | (optional) function to handle scroll |
+| `label` | (optional) label for table |
+| `ariaLabel` | (optional) aria label |
+| `gridBreakPoint` | sizing of how to break up grid for responsiveness |
+| `onSelect` | (optional) function for handling select of tbale |
```

Comment:
possible typo; I think tbale should be table

Reply:
fixed